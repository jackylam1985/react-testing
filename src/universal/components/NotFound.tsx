import * as React from 'react'

export default class NotFound extends React.Component<object, object> {
  render () {
    return (
      <div>
        Not found
      </div>
    )
  }
}
