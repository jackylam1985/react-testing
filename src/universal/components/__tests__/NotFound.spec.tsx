// tslint:disable-next-line no-unused-variable
import * as React from 'react'
import { createRenderer } from 'react-test-renderer/shallow'

import NotFound from '../NotFound'

describe('components/NotFound', () => {

  it('renders', () => {
    const renderer = createRenderer()
    expect(renderer.render(
      <NotFound />,
    )).toMatchSnapshot()
  })
})
