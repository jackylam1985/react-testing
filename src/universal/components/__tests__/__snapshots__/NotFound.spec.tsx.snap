// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/NotFound renders 1`] = `
<Unknown
  status={404}
>
  <div>
    Not found
  </div>
</Unknown>
`;
